### PR TITLE
Add draft release notes for first RC of 1.0.0.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: _releases/1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,805 @@
...
+ * [GUACAMOLE-352](https://issues.apache.org/jira/browse/GUACAMOLE-352) - Add support for dead keys
+ * [GUACAMOLE-447](https://issues.apache.org/jira/browse/GUACAMOLE-447) - Certain keys fail for external keyboards on iOS Safari
+
+### Automatic connection behavior
+
+Guacamole will automatically connect upon login for users that have access to
+only a single connection, skipping the home screen. This feature has been
+tweaked slightly so that it applies only to non-administrative users.  Users
+with access to Guacamole's administrative interface will now see the home
+screen and not automatically connected.
```
Comment (necouchman) [inline _releases/1.0.0.md]: Suggest tweaking this language slightly:
...screen and will not be automatically connected.

File: _releases/1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+default. If known host keys are not provided, Guacamole will continue to behave
+as it has in the past and will allow the connection to succeed, though a
+warning will be logged.
+
+If host keys *are* provided (either in [an `ssh_known_hosts` file within the
+`GUACAMOLE_HOME` directory](/doc/1.0.0/gug/configuring-guacamole.html#ssh-host-verification)
+on the server running guacd, or by passing [the `host-key`
+parameter](/doc/1.0.0/gug/configuring-guacamole.html#ssh-network-parameters) to
+the connection with the key of the specific server), host checking will be
+strictly enforced and the connection will only succeed if the host matches.
```
Comment (necouchman) [inline _releases/1.0.0.md]: Suggested tweak:
...if the host key matches.

File: _releases/1.0.0.md (hunk 1 above)
Comment (mike-jumper) [reply _releases/1.0.0.md]: Good catch. Fixed.

File: _releases/1.0.0.md (hunk 2 above)
Comment (mike-jumper) [reply _releases/1.0.0.md]: Alrighty. Correction made.

**Comment by jmuehlner** [comment]
I don't see anything wrong here! Looks pretty good to me.

**Pull request merged**